src/index.js

```diff
@@ -6451,8 +6451,24 @@ if (! formula && typeof(require) === 'function') {
          * @param integer row number
          * @return string value
          */
-        obj.paste = function(x, y, data) {
+        obj.paste = function(selectedCell, data) {
             // Paste filter
+            selectedCell = selectedCell.map((cell) => parseInt(cell));
```

Comment:
This line is the only difference from https://github.com/jspreadsheet/ce/pull/594.
Upon debugging, I found a singular bug: the expression `selectedCell[2] < selectedCell[0]` evaluates to true when `selectedCell[2]` and `selectedCell[0]` are string values like `"10"` and `"2"`, respectively, due to string comparison.